src/mscorlib/Resources/Strings.resx

```diff
@@ -3596,4 +3654,13 @@
   <data name="SynchronizationLockException_MisMatchedWrite" xml:space="preserve">
     <value>The write lock is being released without being held.</value>
   </data>
-</root>
+  <data name="Memory_ThrowIfDisposed" xml:space="preserve">
+    <value>The memory has been disposed.</value>
+  </data>
+  <data name="Memory_OutstandingReferences" xml:space="preserve">
+    <value>Outstanding references detected that need to be released first</value>
```

Comment:
Nit: This sentence reads strangely to me.  Let's change it to something like "Release all references before disposing this instance." or something like that.  I read this sentence the first time and thought that "outstanding references" were detecting something.